apps/web/src/routes/applet/geom_lin_trans/+page.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+  const modules = import.meta.glob('./**/*.svelte');
```

Comment:
yo what the fuck

Reply:
✨ Vite dust ✨

Reply:
But like, what does it actually do? Import a bunch of svelte components as `modules.ComponentName`?